src/zappy_gui_src/Core/SFML/Map/Tile.cpp

```diff
@@ -39,10 +39,13 @@ Tile Tile::operator=(const Tile &tile)
 
 void Tile::setTexture(const std::string &texturePath)
```

Comment:
Please do not call a function with an already existing name that you use in the same function

Reply:
maybe "setThePathNameOfTheTextureAndSetTheTextureOnTheShapeAndCreateAsharedPointOfTheTexture", but it's too long, so "setTexture" is a good name, doesn't it ?